source/ed/connectors/gtfs_parser.cpp

```diff
@@ -989,7 +1002,11 @@ void StopTimeGtfsHandler::finish(Data& data) {
 }
 
 static int to_utc(const std::string& local_time, int utc_offset) {
-    return time_to_int(local_time) - utc_offset;
+    int local = time_to_int(local_time);
```

Comment:
:+1:
Le 27 juil. 2016 4:28 PM, "pbougue" <notifications@github.com> a écrit :

> In source/ed/connectors/gtfs_parser.cpp
> <https://github.com/CanalTP/navitia/pull/1701#discussion_r72449321>:
>
> > @@ -989,7 +1002,11 @@ void StopTimeGtfsHandler::finish(Data& data) {
> >  }
> >
> >  static int to_utc(const std::string& local_time, int utc_offset) {
> > -    return time_to_int(local_time) - utc_offset;
> > +    int local = time_to_int(local_time);
>
> switched it to -infinite because actual times can be negative after utc
> offset, and also for clarity of debug
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/CanalTP/navitia/pull/1701/files/e38c8fa38b693385a089029cade53a4e26a01c05#r72449321>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADzY8k2O1cxQM0FOcSvV87RjP9zs6EuUks5qZ2r1gaJpZM4JVe07>
> .
>